module.js:327
    throw err;
    ^

Error: Cannot find module 'mocha'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/scratch/dginelli/workspace/repairnator-repairnator-experiments-cettia-cettia-java-server-421051092-20180827-132459-firstCommit/server/src/test/resources/runner.js:4:13)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
                               
[INFO] ------------------------------------------------------------------------
[INFO] Building Cettia Server/Server 1.1.0
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ cettia-server ---
[INFO] Skipping Rule Enforcement.
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ cettia-server ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /scratch/dginelli/workspace/repairnator-repairnator-experiments-cettia-cettia-java-server-421051092-20180827-132459-firstCommit/server/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.2:compile (default-compile) @ cettia-server ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- frontend-maven-plugin:1.6:install-node-and-npm (install node and npm) @ cettia-server ---
[INFO] Node v4.8.3 is already installed.
[INFO] 
[INFO] --- frontend-maven-plugin:1.6:npm (npm install) @ cettia-server ---
[INFO] Skipping execution.
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ cettia-server ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2430 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.2:testCompile (default-testCompile) @ cettia-server ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:2.17:test (default-test) @ cettia-server ---
[INFO] Surefire report directory: /scratch/dginelli/workspace/repairnator-repairnator-experiments-cettia-cettia-java-server-421051092-20180827-132459-firstCommit/server/target/surefire-reports
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/apache/maven/surefire/surefire-junit4/2.17/surefire-junit4-2.17.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/apache/maven/surefire/surefire-junit4/2.17/surefire-junit4-2.17.pom (3 KB at 7.0 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/apache/maven/surefire/surefire-providers/2.17/surefire-providers-2.17.pom
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/apache/maven/surefire/surefire-providers/2.17/surefire-providers-2.17.pom (3 KB at 66.4 KB/sec)
[INFO] Downloading: https://repo.maven.apache.org/maven2/org/apache/maven/surefire/surefire-junit4/2.17/surefire-junit4-2.17.jar
[INFO] Downloaded: https://repo.maven.apache.org/maven2/org/apache/maven/surefire/surefire-junit4/2.17/surefire-junit4-2.17.jar (63 KB at 1185.1 KB/sec)

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running io.cettia.ProtocolTest
10:31:56,211 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
10:31:56,219 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
10:31:56,219 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [file:/scratch/dginelli/workspace/repairnator-repairnator-experiments-cettia-cettia-java-server-421051092-20180827-132459-firstCommit/server/target/test-classes/logback.xml]
10:31:56,356 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
10:31:56,359 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [STDOUT]
10:31:56,382 |-INFO in ch.qos.logback.core.joran.action.NestedComplexPropertyIA - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
10:31:56,436 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [io.cettia] to TRACE
10:31:56,442 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [io.cettia.asity] to INFO
10:31:56,442 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [org.eclipse.jetty] to INFO
10:31:56,442 |-INFO in ch.qos.logback.classic.joran.action.LevelAction - ROOT level set to DEBUG
10:31:56,443 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [STDOUT] to Logger[ROOT]
10:31:56,443 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - End of configuration.
10:31:56,444 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@42e26948 - Registering current configuration as safe fallback point
10:31:56.474 INFO [main] o.e.j.u.log [Log.java:186] Logging initialized @1167ms
10:31:56.604 INFO [main] o.e.j.s.Server [Server.java:327] jetty-9.2.24.v20180105
10:31:57.157 INFO [main] o.e.j.s.h.ContextHandler [ContextHandler.java:744] Started o.e.j.s.ServletContextHandler@65466a6a{/,null,AVAILABLE}
10:31:57.174 INFO [main] o.e.j.s.ServerConnector [AbstractConnector.java:266] Started ServerConnector@73a31e9f{HTTP/1.1}{0.0.0.0:8000}
10:31:57.176 INFO [main] o.e.j.s.Server [Server.java:379] Started @1870ms
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.186 sec <<< FAILURE! - in io.cettia.ProtocolTest
protocol(io.cettia.ProtocolTest)  Time elapsed: 0.815 sec  <<< ERROR!
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:153)
	at io.cettia.ProtocolTest.protocol(ProtocolTest.java:172)

Running io.cettia.support.servlet.HttpSessionResolverTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.172 sec - in io.cettia.support.servlet.HttpSessionResolverTest

Results :

Tests in error: 
  ProtocolTest.protocol:172 Â» Execute Process exited with an error: 1 (Exit valu...

Tests run: 3, Failures: 0, Errors: 1, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Cettia Server ...................................... SUCCESS [  1.021 s]
[INFO] Cettia Server/Server ............................... FAILURE [  7.215 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8.497 s
[INFO] Finished at: 2020-03-18T10:31:58+01:00
[INFO] Final Memory: 16M/67M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project cettia-server: There are test failures.
[ERROR] 
[ERROR] Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-cettia-cettia-java-server-421051092-20180827-132459-firstCommit/server/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project cettia-server: There are test failures.

Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-cettia-cettia-java-server-421051092-20180827-132459-firstCommit/server/target/surefire-reports for the individual test results.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: There are test failures.

Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-cettia-cettia-java-server-421051092-20180827-132459-firstCommit/server/target/surefire-reports for the individual test results.
	at org.apache.maven.plugin.surefire.SurefireHelper.reportExecution(SurefireHelper.java:82)
	at org.apache.maven.plugin.surefire.SurefirePlugin.handleSummary(SurefirePlugin.java:195)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:861)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:729)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :cettia-server
